src/index.html

```diff
@@ -6,9 +6,52 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#HOME">
+        <img
+      class="logo"
+      src="images/logo.png"
+      alt="logo">
```

Comment:
<img width="357" alt="attrCheklist" src="https://github.com/mate-academy/layout_moyo-header/assets/77551820/9124bbfe-b5b7-4f79-9ccb-dddd7f10b967">

**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**